src/main/java/org/testeditor/fixture/web/json/BrowserSettingsManager.java

```diff
@@ -60,86 +57,148 @@
      *          A List of BrowserSetupElement
      */
     protected static List<BrowserSetupElement> getBrowserSettings(String fileName) {
-        
         BrowserSetupReader reader = new BrowserSetupReader();
         List<BrowserSetupElement> elements = reader.readElements(fileName);
-        
         Platform currentPlattform = Platform.getCurrent();
-        checkRedundantElements(elements,currentPlattform);
-        return getBrowserSpecificSetting(elements, currentPlattform);
+        return checkedElementsForDuplicate(elements,currentPlattform);
     }
     
-    protected static void checkRedundantElements(List<BrowserSetupElement> elements, Platform platform) {
-        List<BrowserSetting> duplicateSettings = new ArrayList<>();
-        
+    protected static List<BrowserSetupElement> checkedElementsForDuplicate(List<BrowserSetupElement> elements,
```

Comment:
name is misleading (now) since checking is done but is not the main job of this method, right?